### ISSUE TITLE
automatic installer for forge modpacks crashes

### ISSUE THREAD

**Issue by evschlenz** [cdbbnnyCode/modpack-installer#3]
I could recreate the error multiple times without problem.
I used `python3 install.py "/pathTo/All the Mods 3 - ATM3-v6.1.0.zip"` (Download [here](https://www.curseforge.com/minecraft/modpacks/all-the-mods-3/download/3348062)) to run the installer and got this output:
```
[pack data already unzipped]
Creating .minecraft directory
Creating symlinks
Creating launcher profiles
Installing modloader
Downloading https://files.minecraftforge.net/maven/net/minecraftforge/forge/1.12.2-14.23.5.2855/forge-1.12.2-14.23.5.2855-installer.jar
Status: 200
Exception in thread "main" java.lang.ClassNotFoundException: com.google.common.base.Predicate
        at java.base/java.net.URLClassLoader.findClass(URLClassLoader.java:471)
        at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:589)
        at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:522)
        at java.base/java.lang.Class.forName0(Native Method)
        at java.base/java.lang.Class.forName(Class.java:398)
        at ForgeHack.main(ForgeHack.java:51)
ERROR: Forge did not install correctly!
```
This modpack is for version 1.12.2, so this should normally work, right?

Tested under Ubuntu 21.04 (KDE) with Python 3.9.5 and Java 11 (openjdk 11.0.11 2021-04-20)

**Comment by cdbbnnyCode** [comment]
Thanks for the error log. It looks like a newer version of the Forge installer (even for 1.12.2!) has been released which breaks the automation hack.

**Comment by cdbbnnyCode** [comment]
Fixed in v2.2-beta. If this bug reappears, feel free to reopen this issue.
